components/user-missing-information.tsx

```diff
@@ -0,0 +1,57 @@
+import { ApplicationContext } from "@contexts/application";
```

Comment:
please, use the already existing language functions to translate this modal / page

Reply:
done